manifests/crane/params.pp

```diff
@@ -0,0 +1,10 @@
+# Default params for Crane
+class pulp::crane::params {
+  $ca_cert  = undef
+  $cert = undef
+  $data_dir = '/var/lib/crane/metadata'
```

Comment:
It's the real default for the value, per https://docs.pulpproject.org/plugins/crane/index.html#configuration

If you're using it through pulp class, you probably want it to point to the pulp directory.